src/libostree/ostree-repo-pull.c

```diff
@@ -2589,6 +2589,7 @@ reinitialize_fetcher (OtPullData *pull_data, const char *remote_name, GError **e
  *   * override-url (s): Fetch objects from this URL if remote specifies no metalink in options
  *   * inherit-transaction (b): Don't initiate, finish or abort a transaction, usefult to do mutliple pulls in one transaction.
  *   * http-headers (a(ss)): Additional headers to add to all HTTP requests
+ *   * update-frequency (i): Frequency to call the async progress callback, if any; only values higher than 0 are valid
```

Comment:
You probably want to say "in milliseconds" here, just to make things clear

And why make it "i" if you only allow nonnegative values ?